scss/organisms/_banners.scss

```diff
@@ -472,3 +472,22 @@
 		}
 	}
 }
+
+.jp-wpcom-connect__quest {
+	font-size: $font-title-small;
+
+	&:before {
```

Comment:
Nice CSS solution, but I should have noted in the designs that this should be a [Gridicon](http://automattic.github.io/gridicons/) (`gridicons-notice`).